### PR TITLE
[kodi] Improvements in handling system commands

### PR DESCRIPTION
**Pull request by cweitkamp**
- Improvements in handling system commands - dynamic state options
- Added command for "Quit"
- Fixed to volume update after system has started / connection has been established

Kodi is available as a native application for Android, Linux, Mac OS X, iOS and Windows operating systems. Each platform supports different available system commands (e.g. in a Linux based environment could have rights to reboot the OS whereas an Android application is restricted to quit).

See https://community.openhab.org/t/how-too-close-kodi-on-firestick/46246

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: addons/binding/org.openhab.binding.kodi/README.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ The Kodi thing supports the following channels:
...
...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.kodi/README.md]: The value in the XML starts with capitals, should that be aligned?

File: addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/model/KodiSystemProperties.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    public void setCanreboot(boolean canreboot) {
+        this.canreboot = canreboot;
+    }
+
+    public boolean canSuspend() {
+        return cansuspend;
+    }
+
+    public void setCansuspend(boolean cansuspend) {
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/model/KodiSystemProperties.java]: cansuspend or canSuspend

Comment (martinvw) [reply addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/model/KodiSystemProperties.java]: Should we use a GSON `@serialized` annotation

File: addons/binding/org.openhab.binding.kodi/README.md (hunk 1 above)
Comment (cweitkamp) [reply addons/binding/org.openhab.binding.kodi/README.md]: Yes, nice idea. Thanks. I will change them.

File: addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/model/KodiSystemProperties.java (hunk 2 above)
Comment (cweitkamp) [reply addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/model/KodiSystemProperties.java]: @martinvw Are you talking about the [SerializedName](https://google.github.io/gson/apidocs/com/google/gson/annotations/SerializedName.html) annotation? If not I am afraid I do not know what you like to tell me.

Comment (martinvw) [reply addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/model/KodiSystemProperties.java]: Yes, that one :-)

Comment (cweitkamp) [reply addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/model/KodiSystemProperties.java]: I have to if I want to be compliant with our coding guidelines :wink: .

**Pull request merged**